universal_ssl.go

```diff
@@ -89,3 +98,19 @@ func (api *API) UniversalSSLVerificationDetails(ctx context.Context, zoneID stri
 	}
 	return r.Result, nil
 }
+
+// EditUniversalSSLCertificatePackValidationMethod changes the validation method for a certificate pack
+//
+// API reference: https://api.cloudflare.com/#ssl-verification-ssl-verification-details
+func (api *API) EditUniversalSSLCertificatePackValidationMethod(ctx context.Context, zoneID string, certPackUUID string, setting UniversalSSLCertificatePackValidationMethodSetting) (UniversalSSLCertificatePackValidationMethodSetting, error) {
```

Comment:
let's keep this consistent with the rest of the library

```suggestion
func (api *API) UpdateUniversalSSLCertificatePackValidationMethod(ctx context.Context, zoneID string, certPackUUID string, setting UniversalSSLCertificatePackValidationMethodSetting) (UniversalSSLCertificatePackValidationMethodSetting, error) {
```